docs/src/pages/guides/infinite-queries.md

```diff
@@ -158,11 +158,10 @@ queryClient.setQueryData('projects', data => ({
 Manually removing a single value from an individual page:
 
 ```js
-const newPagesArray = []
-oldPagesArray?.pages.forEach(page => {
-  const newData = page.filter(val => val.id !== updatedId)
-  newPagesArray.push(newData)
-})
+const newPagesArray = oldPagesArray?.pages.map((page) =>
+  page.filter((val) => val.id !== updatedId)
+)
```

Comment:
if we do that and `oldPagesArray` is `undefined`, `newPagesArray` will be `undefined` as well, while in the previous example, it was an empty array. So we'd need `?? []` please

Reply:
Sorry, my bad 😄. I added it.